src/index.js

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
+
+'use strict'
+
+require('dotenv').config()
```

Comment:
I think in this case we don't need `dotenv` any more

Reply:
I think so